v0/destinations/customerio/transform.js

```diff
@@ -144,6 +145,14 @@ function responseBuilder(message, evType, evName, destination, messageType) {
     }
     endpoint = IDENTITY_ENDPOINT.replace(":id", userId);
     requestConfig = defaultPutRequestConfig;
+  } else if (evType === EventType.ALIAS) {
+    // ref : https://customer.io/docs/api/#operation/merge
+    endpoint = MERGE_USER_ENDPOINT;
+    requestConfig = defaultPostRequestConfig;
+    rawPayload.primary = {};
+    rawPayload.primary.id = userId;
+    rawPayload.secondary = {};
+    rawPayload.secondary.id = message.previousId;
```

Comment:
Also, the ID (primary or secondary) is either `id`, `email` or `cio_id`. But here we are relying only on `userId`. Shouldn't we consider them?

Reply:
The matter is, to make the user access cio_id and email, we will need them to input two emails or cio_id's, as the request structure for merge is something like the following:
```
{"primary": {
"email": "cool.person@company.com"
},
"secondary": 
{
"email": "cperson@gmail.com"
}
}
```
To take two emails, we will need to introduce one more specific field. But in case of alias, according to our spec, previousId is already taking care of any previous unique identifier.  So it is more convenient to use only id. WDYT?